crates/rustc_codegen_spirv/src/linker/mod.rs

```diff
@@ -289,6 +295,11 @@ pub fn link(sess: &Session, mut inputs: Vec<Module>, opts: &Options) -> Result<L
             dce::dce(output);
         }
 
+        {
+            let _timer = sess.timer("link_remove_duplicate_lines_2");
+            duplicates::remove_duplicate_lines(output);
+        }
```

Comment:
Rather than running this again, please move the original one here and only run it once. The original is where it is now because none of the later passes could modify function contents, but now with #691, DCE can.